frontend/views/containers/dashboard/SentenceWithMemberTooltip.vue

```diff
@@ -0,0 +1,31 @@
+<template lang="pug">
+component(
+  is='Tooltip'
+  :key='members.length'
+  triggerElementSelector='.t-trigger'
+  direction='bottom-end'
+)
```

Comment:
Is there a reason you are using `<component>` here instead of `<tooltip>` directly? If so, could you add that explanation in a comment?

Reply:
@taoeffect I did leave the explanation comment few lines below here but rephrased it in [this commit](https://github.com/okTurtles/group-income/pull/1468/commits/f1bcc70b31706c703d474a8348ed2d72ac6c820d) to make it more clear.

Reply:
Interesting comment, thank you for adding that! Seems like an advanced usage of Vue!